provisioning_client/inc/azure_prov_client/prov_transport.h

```diff
@@ -59,7 +59,7 @@ extern "C" {
         if (retry_after != NULL)
         {
             // Is the retry after a number
-            if (retry_after[0] >= 0x30 || retry_after[0] <= 0x39)
+            if (retry_after[0] >= 0x30 && retry_after[0] <= 0x39)
```

Comment:
Sorry, this would be good to be covered by an unit test. Many apologies.

Reply:
😿

Reply:
Will be followed up later. Backlog item